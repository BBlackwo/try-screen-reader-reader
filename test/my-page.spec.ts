import { VoiceoverUtil } from './util/voiceover.util';
import { startScreenReader } from 'screen-reader-reader';

const REALLY_BIG_TIMEOUT = 1000000;

describe('my-page', () => {
  it(`should get the output for my-page`, async () => {
    jest.setTimeout(REALLY_BIG_TIMEOUT);

    await page.goto('http://localhost:8080/my-page.html');

    const result = await startScreenReader({ 
      waitForStable: true, 
    });

    VoiceoverUtil.readAllPageContents();

    expect(result).toMatchSnapshot();
  });
});
