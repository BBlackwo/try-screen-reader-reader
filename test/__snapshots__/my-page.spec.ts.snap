// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my-page should get the output for my-page 1`] = `"navigation list  2 items •  bullet link Link 1 •  bullet link Link 2 heading level 1 My Page Thank you for visiting my page"`;
